db/migrations/20230104053806_create_orders_table.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE orders (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  uuid INT NOT NULL,
```

Comment:
```suggestion
  order_number INT NOT NULL,
```
uuid는 기술적 용어입니다. order_number로 바꿔주시고 이는 integer type이 아닌 string입니다. uuid라는건 랜덤의 난수인데 (ex. 123139hjh1jwkgdksaysyf7y2341lhkdlaskfjl123) integer field로는 담을 수 없습니다.